src/groups/group.jl

```diff
@@ -153,6 +153,7 @@ Base.copyto!(e::TE, ::TE) where {TE<:Identity} = e
 Base.copyto!(p, ::TE) where {TE<:Identity} = copyto!(p, e.p)
 Base.copyto!(p::AbstractArray, e::TE) where {TE<:Identity} = copyto!(p, e.p)
 
+Base.isapprox(e1::Identity, e2::Identity; kwargs...) = isapprox(e.group, e1, e2; kwargs...)
```

Comment:
```suggestion
Base.isapprox(e1::Identity{G}, e2::Identity{G}; kwargs...) where {G<:AbstractGroupManifold} = true
```
Could you also add a test for this?

Reply:
Also other new methods in this file seem to lack tests.

Reply:
That was a way trying to reduce ambiguities, that did not work, but I had hoped to have added tests, will check again.

Reply:
I almost gave up reducing ambiguities related to `Identity`. Unless it's an ambiguity in a function call I care about, I just let it be.

Reply:
Then I will remove this just again.